designsafe/static/scripts/applications/controllers/application-tray.js

```diff
@@ -4,6 +4,9 @@ export function applicationTrayCtrl(window, angular, $, _) {
         ['$scope', '$rootScope', '$q', '$timeout', '$uibModal', '$translate', '$state', 'Apps', 'AppsPems', 'AppsSimpleList', 'MultipleList', 'Django', 'toastr', function($scope, $rootScope, $q, $timeout, $uibModal, $translate, $state, Apps, AppsPems, AppsSimpleList, MultipleList, Django, toastr) {
             $scope.tabs = [];
             $scope.simpleList = AppsSimpleList;
+            _.forEach($scope.simpleList.tabs, (tab) => {
```

Comment:
The loop here is used further down in line 220 to populate the contents of each tab object -- we don't want to overwrite the `simpleList.tabs` instance of tabs, since that is used in the `simpleList` itself whenever the tabs are repopulated or refreshed, which happens on every apps update

Reply:
Ah, I see.
The array created here is just the initialization of the object that is populated further down.
Cool, thanks.